app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/IcsExporter.kt

```diff
@@ -47,7 +48,20 @@ class IcsExporter {
                 activity.toast(R.string.exporting)
             }
 
-            outputStream.bufferedWriter().use { out ->
+
+            object : BufferedWriter(OutputStreamWriter(outputStream, Charsets.UTF_8)) {
```

Comment:
Unfortunately I did not find another solution without overriding the `newLine()` method, as I did not want to change the `System.lineSeparator`.